hazelcast/src/main/java/com/hazelcast/jet/impl/JetClientInstanceImpl.java

```diff
@@ -147,14 +147,14 @@ public Job newJobProxy(long jobId, boolean isLightJob, @Nonnull Object jobDefini
     /**
      * For the client side, the jar is uploaded to a random cluster member and then this member runs the main method to
      * start the job.
-     * The jar should have a main method that submits a Pipeline with {@link #newJob(Pipeline)} or
+     * The jar should have a main method that submits a job to Pipeline with {@link #newJob(Pipeline)} or
```

Comment:
```suggestion
     * The jar should have a main method that submits a job with {@link #newJob(Pipeline)} or
```

Reason: job is not submitted to a pipeline, but Pipeline is an argument to submission. I think it's redundant to mention Pipeline, as it's already in the method signature you are referencing to.

Reply:
Rev : cab8bc17a8721ca01d065a3e36545903bc121fb3